app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -311,6 +317,15 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
                     updateOrDeleteWebViewPreview()
                 }
             }
+
+            @OnLifecycleEvent(Lifecycle.Event.ON_START)
+            fun onStart() {
+                if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)) {
+                    WebSettingsCompat.setForceDark(webView!!.settings,
+                            if (settingsDataStore.darkMode) WebSettingsCompat.FORCE_DARK_ON
+                            else WebSettingsCompat.FORCE_DARK_OFF)
+                }
```

Comment:
Don't force the `webView!!`.
Also this 
`if (settingsDataStore.darkMode) WebSettingsCompat.FORCE_DARK_ON else WebSettingsCompat.FORCE_DARK_OFF`
should be delegate to the viewModel `viewModel.style`
This too `WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)`
`viewModel.isDarkModeSupported`